assimilation_code/programs/obs_diag/threed_sphere/obs_diag.rst

```diff
@@ -536,10 +536,10 @@ Please note:
 
 +-----------+----------------------------------------------------------------------------------------------------------+
 | |image3|  | `Instructions for viewing the rank histogram with                                                        |
-|           | ncview <http://www.image.ucar.edu/DAReS/DART/DART2_Documentation.php#ncview_histogram>`__.               |
+|           | ncview <https://cirrus.ucsd.edu/ncview/>`__.                                                             |
 +-----------+----------------------------------------------------------------------------------------------------------+
 | |image4|  | `Instructions for viewing the rank histogram with                                                        |
-|           | Matlab <http://www.image.ucar.edu/DAReS/DART/DART2_Documentation.php#mat_obs>`__.                        |
+|           | Matlab <https://docs.dart.ucar.edu/en/latest/guide/matlab-observation-space.html>`__.                    |
```

Comment:
this is the right page to link to, but to future proof this it is better to link to the section or anchor. 

That way if the documentation is rearranged, or if the documentation is built as pdf, the link goes to the section rather than hardcoded to this webpage